router-tests/websocket_test.go

```diff
@@ -197,9 +199,9 @@ func TestWebSockets(t *testing.T) {
 				// Trigger the subscription via NATS
 				subject := "employeeUpdated.3"
 				message := []byte(`{"id":3,"__typename": "Employee"}`)
-				err := xEnv.NC.Publish(subject, message)
+				err := xEnv.NatsConnectionBySourceName[defaultSourceName].Publish(subject, message)
```

Comment:
I think we will only ever have 2 nats instances in the testenv.
Could we make this static and just add the two instances as 2 fields instead of having to pick from a map, which might lead to errors by itself?

Reply:
I thought about this, but I added some foundations for supporting however many sources we want, in case we want to extend with, for example, Kafka, or whatever.

I can make it explicit fields if you prefer, though.

Reply:
I understand, this makes sense, although each additional client will require the client setup anyways.
So I'm not sure what we win through having a map.

Reply:
I'll change it to explicit fields 👍